cli/cmd/upgrade.go

```diff
@@ -0,0 +1,169 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+	"reflect"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/timescale/tobs/cli/pkg/utils"
+)
+
+// upgradeCmd represents the upgrade command
+var upgradeCmd = &cobra.Command{
+	Use:   "upgrade",
+	Short: "Alias for helm upgrade",
+	Args:  cobra.ExactArgs(0),
+	RunE:  upgrade,
+}
+
+func init() {
+	rootCmd.AddCommand(upgradeCmd)
+	addHelmInstallFlags(upgradeCmd)
+	upgradeCmd.Flags().BoolP("reset-values", "", false, "Reset helm chart to default values of the helm chart. This is same flag that exists in helm upgrade")
+	upgradeCmd.Flags().BoolP("reuse-values", "", false, "Reuse existing release values do not override them with modified default chart versions. This is same flag that exists in helm upgrade ")
```

Comment:
didn't we want this to default true?

Reply:
By default, it's true if no `--set` or `-f` flags are used.
![Screenshot 2020-12-17 at 10 46 23 PM](https://user-images.githubusercontent.com/25104868/102520676-f183b480-40b9-11eb-963f-57e6f650dce5.png)

For reference on helm upgrade flags. This [blog](https://medium.com/@kcatstack/understand-helm-upgrade-flags-reset-values-reuse-values-6e58ac8f127e) is quite good